utils/run_test.sh

```diff
@@ -131,6 +131,7 @@ if [ -d "/mnt/daos" ]; then
     run_test "${SL_PREFIX}/bin/vos_size"
     run_test "${SL_PREFIX}/bin/vos_size.py" \
              "${SL_PREFIX}/etc/vos_size_input.yaml"
+    run_test "${SL_PREFIX}/bin/vos_size_dfs_sample"
```

Comment:
I think we should update the vos README to give an example for how to use the result.  There is a section there that may actually reference the old file you removed.

Reply:
The old file “vos_dfs_sample.yaml” is not mentioned on the VOS [README](https://github.com/daos-stack/daos/blob/master/src/vos/README.md) file. I totally agree that it should be documented. I will update the documentation.